src/ducks/apps/components/ApplicationPage/index.jsx

```diff
@@ -25,7 +26,8 @@ import withBreakpoints from 'cozy-ui/transpiled/react/helpers/withBreakpoints'
 const MOBILE_PLATFORMS = ['ios', 'android']
 const isMobilePlatform = name => MOBILE_PLATFORMS.includes(name)
 
-const { BarCenter } = cozy.bar
+// In case we are in an Intent, `cozy.bar` is undefined
```

Comment:
```suggestion
// In case we are in an Intent, `cozy.bar` is undefined and it's not a big deal since we don't need the cozy-bar to be displayed on an intent
```

Reply:
Merci, c'est corrigé